defaults.go

```diff
@@ -274,6 +274,11 @@ func (t Type) String() string {
 // String returns the string representation for the class c.
 func (c Class) String() string {
 	if c1, ok := ClassToString[uint16(c)]; ok {
+		// Special case ANY, even though it's in the map we should not translate CLASS255 to ANY.
+		if c == 255 {
```

Comment:
Better still (cf. [section 3.2 of RFC 2929](https://tools.ietf.org/html/rfc2929#section-3.2)):
```go
// Output mnemonics only for ranges not reserved by RFC 2929.
// This could also exclude 0 and 0xFFFF ("assignment requires an IETF Standards Action"),
// and maybe also 0xFF00 through 0xFFFE ("Private Use").
if c < 128 || c > 255 {
	if c1, ok := ClassToString[uint16(c)]; ok {
		return c1
	}
}
return "CLASS" + strconv.Itoa(int(c))
```

And we should also to the same for Type.

Reply:
But wouldn't reserving by the IETF mean we put it in this ClassToString? The private range might bite people extending this map with a private range class?

What are you looking for, for types? Note that we actually sport a `ANY` RR? /me ducks

Reply:
RFC 2929 reserves CLASS values 128 through 255 "as QCLASSes only" (with only two actually assigned: 254 NONE and 255 ANY), which means that any value in that range will never be valid in a zone file. Similar but weaker reasoning applies to 0 and 0xFFFF, and I'm not sure about the private use range, but I guess I'd trust users to only add to the maps if they really want the values treated as standard.

But maybe none of that really matters, because all we really want is to emit strings that we can read back. Updated suggestion:
```go
func (c Class) String() string {
	if s, ok := ClassToString[uint16(c)]; ok {
		// Only emit mnemonics when they are unambiguous.
		if _, ok := StringToType[s]; !ok {
			return s
		}
	}
	return "CLASS" + strconv.Itoa(int(c))
}
```